src/sentry/api/base.py

```diff
@@ -497,14 +498,16 @@ def resolve_region(request: Request):
 class EndpointSiloLimit(SiloLimit):
     def modify_endpoint_class(self, decorated_class: Type[Endpoint]) -> type:
         dispatch_override = self.create_override(decorated_class.dispatch)
-        return type(
+        new_class = type(
             decorated_class.__name__,
             (decorated_class,),
             {
                 "dispatch": dispatch_override,
                 "__silo_limit": self,  # For internal tooling only
```

Comment:
I think we can just drop the leading underscores. It's pretty much guaranteed that we'll need this for the API Gateway

Reply:
Sounds okay to me, but let's change that in a separate PR.